lib/settlement/driip-settlement.spec.js

```diff
@@ -495,7 +495,10 @@ describe('Driip settlement operations', () => {
             nonce: 1,
             amount: '100',
             currency: {ct: address0, id: 0},
-            sender: {wallet: ''},
+            sender: {
+                wallet: '',
+                data: Buffer.from(JSON.stringify({ref: ''})).toString('base64')
```

Comment:
Do you think it would be a a good idea to create a helper function for this? I've seen this being done a couple of times in the PR already.

Reply:
Could be extracted into the test-utils 👍